packages/dev/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@clerk/dev",
```

Comment:
Should we change this to `@clerk/clerk-dev` so that PR titles are a little more descriptive? They'd be `feat(clerk-dev)` as opposed to `feat(dev)`.

Reply:
Great idea! Fixed in 592b4cd118e7e40117a1c0814072393593180b36